components/card/metadata/card.yml

```diff
@@ -32,10 +33,11 @@ examples:
               </div>
             </div>
             <div class="spectrum-Card-content">
-              <div class="spectrum-Card-subtitle spectrum-Detail spectrum-Detail--sizeS">jpg</div>
+              <div class="spectrum-Card-description">Optional description should be kept to one or two lines</div>
             </div>
           </div>
           <div class="spectrum-Card-footer">
+            <hr class="spectrum-Divider spectrum-Divider--sizeS">
```

Comment:
Instead of a divider component, should we take a similar tact of extending the styles from divider as a footer border-top maybe?

Reply:
That was what I was doing originally but was told by design that this is supposed to be a Divider component...